### ISSUE TITLE
Weighted mask inverted from what is expected

### ISSUE THREAD

**Issue by theahura** [moodoki/semantic_image_inpainting#6]
If I understand the paper correctly, the input ought to weight pixels around the mask higher than those farther from the mask. 

- gen_mask() creates a binary mask of 0s and 1s, with 0s being the masked out portion. 

- createWeightedMask() convolves over the binary mask. This means that the pixels closest to the masked out portion will have have a LOWER weight than those that are farther away. 

- the loss contextual loss defined in build_inpaint_graph does not invert these weights. Rather, it multiplies the mask with the generated output and with the real data and subtracts the two. This means that the contextual loss is based more heavily on the differences between pixels that are farther from the mask itself. 

I could be misinterpreting code or the paper somewhere. However visualizing these masks seems to confirm my suspicion. Please advise!

**Comment by moodoki** [comment]
That issue shold be fixed here. Please use this branch (extensions) instead of master. 
https://github.com/moodoki/semantic_image_inpainting/tree/extensions

**Comment by elham1992** [comment]
Hi, Thanks for sharing your implementation.
i programing dcgan with keras, and save model ,then Conversion from h5py to Tensorflow ProtoBuf format ,and use your code.
 i  have a problem, my program dosen't run carrectly and shows error like in the below.

" keyerror:"the name 'dcgan/z:0' refers to a tensor which does not exist'"
 please help me to solve it , thanks.
